### PR TITLE
fixes for ubuntu installator

### PR DESCRIPTION
**Pull request by paulperegud**

### CODE REVIEW

File: Installer/Installer_Linux/install.sh [hunk 1]
```diff
@@ -171,10 +193,10 @@ function install_golem()
 # @brief Main function
 function main()
 {
-    # Make sure only root can run our script
+    # Make sure user calls this script using sudo
     if [[ $EUID -ne 0 ]]; then
-        ask_user "This script need sudo access. Do you wan to continue? (y/n)"
-        [[ $? -eq 1 ]] && exec sudo bash "$0" || return 1
+        error_msg "This script needs to be called with sudo. E.g. sudo ./install.sh"
```
Comment (banasiakadam60) [inline Installer/Installer_Linux/install.sh]: Why did you remove this? It was asking user if he want to run script as root and if he want, then rerun script with `sudo`. What's wrong with this?

Comment (paulperegud) [reply Installer/Installer_Linux/install.sh]: I've misread part that restarted the script. Rolling this back.
